repos/system_upgrade/common/actors/missinggpgkeysinhibitor/libraries/missinggpgkey.py

```diff
@@ -221,23 +224,26 @@ def _get_repo_gpgkey_urls(repo):
     Returned gpgkeys are URLs with already expanded variables
     (e.g. $releasever) as gpgkey can contain list of URLs separated by comma
     or whitespaces.
+    If no gpgcheck=0 is present in the repo file, None is returned. If the
+    gpgcheck is missing or enabled and no gpgkey is present, empty list is
+    returned, which means the repo can not be checked.
     """
 
+    if not repo.additional_fields:
+        return []
+
     repo_additional = json.loads(repo.additional_fields)
 
     # TODO does the case matter here?
-    if 'gpgcheck' in repo_additional and repo_additional['gpgcheck'] in ('0', 'False', 'no'):
+    if 'gpgcheck' in repo_additional and repo_additional['gpgcheck'] in (0, '0', 'False', 'no'):
```

Comment:
@Jakuje is there a case when you could find the `0` (int) instead of the string? I thought that json.loads will return the string always in this case. Just want to be sure.

Reply:
Probably just bad test data in the unit/component test. Will remove.